### ISSUE TITLE
Add Aria ignored elements to non-navigable/non-informational items on the progress and rankings view

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1111]
* ~~Skilltree logo triggers multiple elements to be read when using a screen reader, should only trigger one~~ obe
* ~~Settings menu reads as "Settings Menu Button Menu" could be improved~~ obe
* ~~Help Menu rads as "Help Menu Button Menu" could be improved~~ obe
* ~~Usage chart link reads as "m dash bullet usage"~~ obe
* ~~several of the separators that are only used for visual layout should be aria-ignored~~ obe
* the graphs displayed should probably ignore the progress bar/radial gauge and provide an aria-label that describes the percentage as xx percent complete - apex charts doesn't support better labeling

**Comment by mosefer** [comment]
PR #1375

**Comment by rmmayo** [comment]
:+1:
